### PR TITLE
Feature: use classification as tags

### PR DESCRIPTION
**Pull request by TrueBrain**
This is mostly for backwards compatibility; as we no longer allow custom tags, the best-next-thing are the classification markers.

This means in current OpenTTD clients they show up as tags, in which you can search etc. This was kinda how tags were meant, but sadly people decided to use them for completely different purposes.

### CODE REVIEW

File: bananas_server/index/local.py [hunk 1]
```diff
@@ -146,8 +146,14 @@ def _read_content_entry_version(self, content_type, unique_id, data, md5sum_mapp
         size += len(md5sum) + 2
         size += len(dependencies) * 4
         size += 1
-        for tag in data.get("tags", []):
-            size += len(tag) + 2
+        for key, value in data.get("classification", {}).items():
+            size += len(key) + 2
```
Comment (frosch123) [inline bananas_server/index/local.py:150]: This does not match _send_content_entry

Comment (TrueBrain) [reply bananas_server/index/local.py:150]: Yup. And yes, this is fine ;)

This already checks for the future, that we can always send `{key}:{value}` or `{key}:yes`. The bananas-server implementation also checks if this fits. This way, nobody will introduce a package where we cant send key/value, if we ever want to send the full classification to the client.

In reality, it currently only sends either `key` or `value`, which is always smaller than the combination.

In other words: it does match the validation of bananas-server, and it always sends something smaller as tags.

**Review by frosch123 — APPROVED** [review]
